spec/admin/contribution_spec.rb

```diff
@@ -0,0 +1,20 @@
+require 'spec_helper'
+
+describe Contribution do
+  describe 'Contribution' do
+    let(:resource_class) { Contribution }
```

Comment:
You don't need this `let` here, when you need to get the class being tested you can invoke the method `described_class` from RSpec.